### PR TITLE
Properly append port to URL when needed.

### PR DESCRIPTION
**Pull request by DFurnes**
I forgot I had made this change locally until I did a `rm -rf node_modules && npm install`, whoops!

#### Before
screen shot 2017-03-03 at 12 29 49 pm

😭 😭 😭 

##### After
screen shot 2017-03-03 at 12 30 35 pm

😌 😌 😌

### CODE REVIEW

File: src/RestApiClient.js [hunk 1]
```diff
@@ -74,7 +74,8 @@ class RestApiClient {
    * @return {String}
    */
   getLocalUrl() {
-    return `//${window.location.hostname}${window.location.port}`;
+    const port = window.location.port ? `:${window.location.port}` : '';
```
Comment (weerd) [inline src/RestApiClient.js:77]: that's weird. I had checked this and `window.location.port` would default to empty string already if nothing was assigned to it...

Comment (DFurnes) [reply src/RestApiClient.js:77]: It does! But we need to append the `:` at the beginning if it's not an empty string.

Comment (weerd) [reply src/RestApiClient.js:77]: oh doi! hehe

**Review by weerd — COMMENTED** [review]
👍

File: src/RestApiClient.js (hunk 1 above)
Comment (weerd) [reply src/RestApiClient.js:77]: I just prefer port-less dev domains ;)

**Pull request merged**